lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1894,6 +1894,7 @@ void MtohRenderOverride::_PickByRegion(
     pickParams.viewMatrix.Set(viewMatrix.matrix);
     pickParams.projectionMatrix.Set(adjustedProjMatrix.matrix);
     pickParams.collection = _renderCollection;
+    pickParams.collection.SetExcludePaths(_wireframeSelectionHighlightSceneIndex->GetSelectionHighlightMirrorPaths());
```

Comment:
We need to exclude selection highlight prims. If we don't, selecting the highlight mirror of a mesh that is trimmed by a GeomSubset selection ends up selecting the original mesh, instead of selecting the GeomSubset.